src/CommandHandler.h

```diff
@@ -20,90 +20,91 @@ struct StreamSettings {
   std::string ConfigStreamJson;
 };
 
-/// \brief Helper for adding more error information on parse error.
+
+/// \brief If fails to parse the `Command`, adds error info and throws
+/// exception.
+///
+/// \param  Command Command passed to the program.
+///
+/// \return If parsing successful returns `nlohmann::json`, otherwise throws an
+/// exception.
 nlohmann::json parseOrThrow(std::string const &Command);
 
-/// \brief Interprets and execute commands received.
+std::string findBroker(std::string const &);
+
+/// Formats exceptions into readable form.
```

Comment:
Same as above, although you could remove the one in the code and just keep the documentation in the header